src/Tools/Test Framework/Test Runner/src/TestSuiteMgt.Codeunit.al

```diff
@@ -309,6 +309,34 @@ codeunit 130456 "Test Suite Mgt."
         GetTestMethods(ALTestSuite, CodeunitMetadata);
     end;
 
+    /// <summary>
+    /// Converts a text representation of a TestType to an option.
+    /// </summary>
+    /// <param name="TestType">Text representation of TestType, specified from CLI</param>
+    /// <returns>One of the options from TestType field in CodeunitMetadata table, except for None</returns>
+    internal procedure ConvertTestTypeToOption(TestType: Text) result: Option None,UnitTest,IntegrationTest,Uncategorized;
```

Comment:
See the comment on integer. If we have this in the platform on the Codeunit metadata, we can easly convert instead of hardcoding.